cmd/translate/main_grpc_integration_test.go

```diff
@@ -672,6 +674,272 @@ func Test_ListTranslations_gRPC(t *testing.T) {
 	}
 }
 
+func Test_UpdateTranslationFromMask_gRPC(t *testing.T) {
+	t.Parallel()
+
+	ctx, subtest := testutil.Trace(t)
+
+	// Prepare
+	langs := rand.Languages(5)
+
+	service := createService(ctx, t)
+	createTranslation(ctx, t, service.GetId(), &translatev1.Translation{
+		Original: false,
+		Language: langs[1].String(),
+		Messages: []*translatev1.Message{
+			{
+				Id: "Hello", Message: "Sveiks",
+			},
+		},
+	})
+
+	service2 := createService(ctx, t)
+	createTranslation(ctx, t, service2.GetId(), &translatev1.Translation{Original: false, Language: langs[2].String()})
+
+	service3 := createService(ctx, t)
+	createTranslation(ctx, t, service2.GetId(), &translatev1.Translation{Original: false, Language: langs[3].String()})
+
+	service4 := createService(ctx, t)
+	createTranslation(ctx, t, service2.GetId(), &translatev1.Translation{Original: false, Language: langs[4].String()})
+
+	service5 := createService(ctx, t)
+	service6 := createService(ctx, t)
+	service7 := createService(ctx, t)
+	service8 := createService(ctx, t)
+
+	tests := []struct {
+		request     *translatev1.UpdateTranslationRequest
+		translation *translatev1.Translation
+		name        string
+	}{
+		{
+			name: "Update messages field in original translation",
```

Comment:
Could you group these tests? 
Right now they are a. bit mixed up.
Something like -
update message/s for translation, 
update message/s for original, 
update original status for translation